troubleshooting/index.md

```diff
@@ -39,10 +39,10 @@ If you’re seeing this message, the page you’d like to use GhostText on doesn
 
 If you’re seeing this message, the browser extension can’t find the GhostText server running in an editor. Ensure that:
 
-- Your editor is open
-- Its GhostText extension is installed; Install it from the [list of supported editors](/#installation) if you haven’t already.
-- The GhostText server is running; In most editor extensions this is started automatically.
-- The server port matches; It’s 4001 by default, it can be changed in the options.
-- There are no other servers using the port. For example what happens when you open http://localhost:4001? The page should return details about WebSocket and the editor should start a new connection.
+- Your editor is launched
```

Comment:
```suggestion
- Your editor is open
```

Reply:
It may not make much sense but here's my reason: there is difference between a _opened_ and _launched_ application. The former is more about specific window/file while the latter is more focused on the program itself. For example, on macOS it's common to have a editor _launched_ with no _open_ window.

But it doesn't really matter so I can revert it if you want.

Reply:
The correct term here would be "running", but "open" is the most common

- Microsoft uses open/launch interchangeably
	https://support.microsoft.com/en-us/account-billing/how-to-open-microsoft-store-on-windows-10-e080b85a-7c9e-46a7-8d8b-3e9a42e32de6
	
- macOS refers to them as "open", regardless of windows

	<img width="780" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/140728617-c2a0eb66-fec6-4536-a2e4-0b3eecb38c64.png">